README.md

```diff
@@ -129,13 +130,20 @@ Triggered when the client is successfully connected to the conference after call
 Triggered when the client is disconnected from the conference by any means; in particular, by the [Disconnect](#disconnect) function.
 
 ### On Participant Added
-Triggered when a remote participant is added to the conference. Provides information about the participant in its argument.
+Triggered when a remote participant is added to the conference. Provides the participant status and participant info in its argument.
 
 Example:  
-![example](Images/on_participant_added.PNG)
+![example](./Images/on_participant_added.PNG)
 
-### On Participant Left
-Triggered when a remote participant leaves the conference. Provides the participant ID in its argument.
+### On Participant Updated
+Triggered when a remote participant's status is updated. Provides the participant status and participant info in its argument.
+
+Example:  
+![example](Images/on_participant_updated.png)
+
+The following graphic shows possible status changes:
+
+![example](./Images/participant_status-changes.png)
```

Comment:
I'm just leaving this note for conscience reasons to say I do not agree with the need to extend these events while also seemingly needing to explain all this detail for the user to understand it.